finetuner/__init__.py

```diff
@@ -347,6 +326,9 @@ def get_model(
     """
     from commons.data.inference import ONNXRuntimeInferenceEngine
 
+    for key in logging.root.manager.loggerDict:
+        logging.getLogger(key).setLevel('CRITICAL')
```

Comment:
why are you silencing all loggers?

Reply:
I thought that we only want to log the messages of the executor which are set by the ONNXRuntimeInferenceEngine. However, I can also change it to only silence the commons and the ulrlib logger in the case we want to log more. However, this means that there might occur other unexpected log messages after some changes.

Reply:
try to silence only these loggers first, and see how it looks